### PR TITLE
Jia/QUILL-1816/remove-reset.css

### PR DESCRIPTION
**Pull request by jia-deriv**

### CODE REVIEW

File: lib/components/Notification/notification.scss [hunk 1]
```diff
@@ -1,4 +1,17 @@
...
+    }
+    &:visited {
+        text-decoration: none;
+    }
+    &:hover {
+        text-decoration: none;
+    }
+    &:active {
+        text-decoration: none;
+    }
```
Comment (prince-deriv) [inline lib/components/Notification/notification.scss:13]:
```suggestion
    &:link,  &:visited ,  &:hover,   &:active {
        text-decoration: none;
     }
```

File: lib/styles/index.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /* Global Styles */
-@import "./reset.scss";
 @import "@deriv-com/quill-tokens/dist/quill.css";
+@import "../../node_modules/modern-normalize/modern-normalize.css";
```
Comment (prince-deriv) [inline lib/styles/index.scss:3]: Maybe we can import the unpkg or CDN version of this instead?

Comment (jia-deriv) [reply lib/styles/index.scss]: Sure I'll use unpkg for this

File: package.json [hunk 3]
```diff
@@ -148,6 +148,7 @@
         "@deriv/quill-icons": "^1.22.10",
         "@headlessui/react": "1.7.18",
         "dayjs": "^1.11.11",
+        "modern-normalize": "^2.0.0",
```
Comment (prince-deriv) [inline package.json:151]: do we still need this?

Comment (jia-deriv) [reply package.json]: I missed it. Will remove it.

**Review by prince-deriv — APPROVED** [review]

**Pull request merged**